docs/shared/asm-ingress-gateway/autoscaling-v2beta1.yaml

```diff
@@ -0,0 +1,19 @@
+# Optional: HorizontalPodAutoscaler will automatically scale the gateway replica count based on
+# CPU utilization
+# Used on <1.23 K8S versions
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: istio-ingressgateway
+spec:
+  maxReplicas: 5
+  metrics:
+  - resource:
+      name: cpu
+      targetAverageUtilization: 80
+    type: Resource
+  minReplicas: 3
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: istio-ingressgateway
```

Comment:
**Question:**
Is there a reason for not adding this `HorizontalPodAutoscaler` to `docs/ingress-gateway-asm-manifests/base/`?

Reply:
Just found my answer:
It's because we already have a `HorizontalPodAutoscaler` in `docs/ingress-gateway-asm-manifests/base/`: https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples/blob/8413b57f16b9c14fde51b7e396a1408dc2045841/docs/ingress-gateway-asm-manifests/base/deployment-service.yaml#L88
Good catch! ⭐